### ISSUE TITLE
Mobile dApp Tutorial (Medium article) - Reward $2k (in $CCDs) [+ Video Tutorial Bonus]

### ISSUE THREAD

**Issue by bogacyigitbasi** [Concordium/concordium-bounty-program#3]
- Mobile dApp using walletconnect to sign a sponsored transaction, mint, transfer, and view NFTs. (React Native or Flutter)
- Written tutorial on Medium and a video for the bonus. 
- Tutorial documentation must show all details of the implementation with brief descriptions. 
- Open-sourced Github repo
- Hosted on testnet, available on Google Play store and TestFlight for iOS.  
- You should use Concordium tools like [VSCode extension](https://marketplace.visualstudio.com/items?itemName=Concordium.concordium-smart-contracts),[ccd-js-gen](https://www.npmjs.com/package/@concordium/ccd-js-gen), [sc-tools](https://sctools.mainnet.concordium.software/)) if applicable
- If maintenance is required, the participant is responsible for 3 months.

**Comment by hoepeyemi** [comment]
Gm, This looks cool. Can i take it up?

**Comment by bogacyigitbasi** [comment]
Be my guest!

**Comment by hoepeyemi** [comment]
Can you clarify on what you meant by sponsored transaction?

**Comment by bogacyigitbasi** [comment]
Here is the standard's document: https://proposals.concordium.software/CIS/cis-3.html
An example smart contract: https://github.com/Concordium/concordium-rust-smart-contracts/tree/main/examples/cis3-nft-sponsored-txs
And an example dApp using it: https://github.com/Concordium/concordium-dapp-examples/tree/main/sponsoredTransactions

**Comment by hoepeyemi** [comment]
These links gave me clarity.

**Comment by hoepeyemi** [comment]
Thanks, One last thing. I am seeing $2k in the bounty list and i am seeing $1.5k as the title here. can you clarify on that pls?

**Comment by bogacyigitbasi** [comment]
Copy paste error, sorry. The bounty list is the right one, which is $2k :)

**Comment by hoepeyemi** [comment]
Thanks. Is there any other information i need to know or can i start to build?

**Comment by hoepeyemi** [comment]
Hi, Did you got my question?

**Comment by bogacyigitbasi** [comment]
What question? About starting? Please get started any time you want, and reach out to us through community managers once you have something ready for a feedback review

**Comment by josidbobo** [comment]
Hello I will be taking this up.

**Comment by josidbobo** [comment]
Is this still open I’ve made some progress on it, where do I make a PR to? @bogacyigitbasi

**Comment by bogacyigitbasi** [comment]
Hey, yes it's still open. Could you please create a PR on your personal repo where we can leave comments including a readme with links to articles, video, and link to the hosted app :)

**Comment by josidbobo** [comment]
The article should discuss steps in recreating the mobile app only or the contract and the app together? @bogacyigitbasi

**Comment by bogacyigitbasi** [comment]
Yes, it must be a beginner-friendly, step-by-step guide that shows how to do it as a complete tutorial. If you used the smart contract without modification from examples repo, it is okay not to explain everything there. But it would be great if you could show the function to be shared from the contract first and the SDK implementation to call it including the JSON parameter etc. Hope its clear?@josidbobo If you could send me a draft version I would be happy to give me 5 cents and then you can iterate.

**Comment by josidbobo** [comment]
It’s very clear, do I send the draft to you email for feedback first?

 I also need the Concordium RPC url for mainnet & testnet as well as chain Id so I integrate with Walletconnect @bogacyigitbasi

**Comment by bogacyigitbasi** [comment]
Hey @josidbobo we use gRPC and the testnet endpoint `node.testnet.concordium.com` and the port is `20000` and chain ID should be the following ccd:4221332d34e1694168c2a0c0b3fd0f27 from [walletconnect](https://docs.walletconnect.com/cloud/chains/chain-list) docs. When you have the draft on Medium you can share it here or send me the link from telegram. Let me know if there is an issue.

**Comment by iamskeva** [comment]
Hi @bogacyigitbasi I will be working on this bounty. will give you feedback in the coming week

**Comment by bogacyigitbasi** [comment]
As the other two volunteers have been silent for a long time, I don't see any reason to wait. LFG! 🚀

**Comment by onyekachi11** [comment]
Hi @bogacyigitbasi hw do i get on the testflight for ios?

**Comment by binbash0** [comment]
@bogacyigitbasi Is this program taken? Or I can start? I can get it done later this month...

**Comment by bogacyigitbasi** [comment]
The program is closed, wait for the next season @binbash0, watch out our socials/telegram channels. New ones will be announced.

**Comment by binbash0** [comment]
Okay can't wait...
